cmd/thanos/sidecar.go

```diff
@@ -362,7 +362,7 @@ func runSidecar(
 			s := shipper.New(logger, reg, conf.tsdb.path, bkt, m.Labels, metadata.SidecarSource,
 				uploadCompactedFunc, conf.shipper.allowOutOfOrderUpload, metadata.HashFunc(conf.shipper.hashFunc), conf.shipper.metaFileName)
 
-			return runutil.Repeat(30*time.Second, ctx.Done(), func() error {
+			return runutil.RepeatWithJitter(30*time.Second, ctx, 0.2, func() error {
```

Comment:
Does this really help though if we are jittering only 6s; at least for us block uploads can take far longer

Reply:
We can increase the jitter. What value do you think that makes sense?

Reply:
I don't know, the whole method feels kinda weird. It was about in house blob storage and blocks being uploaded at same time right?